frontend/ui/widget/buttondialog.lua

```diff
@@ -128,7 +134,7 @@ function ButtonDialog:init()
             bordersize = 0,
             TextBoxWidget:new{
                 text = self.title,
-                width = content_width - 2 * (title_padding + title_margin),
+                width = self.buttontable.width - 2 * (title_padding + title_margin),
```

Comment:
Is that change because the static `content_width` is no more valid and you need to use the possibly recomputed (shrinked?) `self.buttontable.width` ?
If yes, best to add a `function ButtonTable:getWidth() return self.width end` so it's clear it may not be the one provided as `ButtonTable:new{ width= }`.